### ISSUE TITLE
Display CA certificates somewhere in the web app

### ISSUE THREAD

**Issue by fhunleth** [nerves-hub/nerves_hub_web#348]
**Describe the bug**
CA certificates are important for device auto-registration, but they currently only show up in the CLI

**To Reproduce**
Add a bunch of CA certificates with `mix nerves_hub.ca_certificate register ...`. Go to the web UI.

**Expected behavior**
It's possible to list out CA certificates. It would be really useful to see how many times they were used and their expiration dates.

**Comment by pcmarks** [comment]
Just want to make sure: the database stores three types of certificates: ca_certificates, device_certificates and user_certificates. This issue is referring to **ca_certificates**, right?

**Comment by fhunleth** [comment]
Yes. 

Also, the Web UI already displays user certificates. Device certificates might be interesting to have, but they'd naturally go under the device informational screen.

**Comment by pcmarks** [comment]
On a question of naming. 

1. Since ca certificates are tied to organizations, it would seem that the table `ca_certificates` could be renamed `org_certificates`. This is similar to the `user` and `user_certificate` tables. This would require adding some Ecto migrations, etc., however.

2. Leave the ca certificate table name the same but the Ecto module would be called `org_certificate` and in the code implement `org_certificate_controller`, `org_certificate_view`, etc.

I favor 2. 

Thoughts?

**Comment by fhunleth** [comment]
I feel uneasy changing the name. `user_certificate`s are used to authenticate users. `ca_certificates` are not used to authenticate organizations. `ca_certificates` are used the first time devices connect to authenticate their device certificates.

**Comment by fhunleth** [comment]
I might suggest `device_ca_certificate`, but that's getting long to type.

**Comment by pcmarks** [comment]
Thanks for the clarification.

It turns out that ca certificates are already accounted for under devices, e.g., `NervesHubWebCore.Devices.CACertificates` So that can stay the same but there will be a ca_certificate controller, etc. that is reached via the organization settings page - if that makes sense. 

If you want, I can sketch out how the UI will behave and look like tomorrow.

I've already made the changes that allow one to see the ca certificates associated with an organization. It looks and works like the `User Certificates` button on the user settings page.

**Comment by fhunleth** [comment]
That makes sense.  It sounds like you're putting the certificate list in the right place, so I don't think it's necessary for us to review how the UI behaves. If you're unsure, feel free to let us know. However, we did recently get an offer for help with the UI. I'm quite happy to postpone UI perfection for their help.

**Comment by pcmarks** [comment]
The User Certificate page has new (needs an icon) and delete buttons. The delete button works but the new doesn't - error. I'll create an issue.

Should THE Device (CA) Certificate page have operational new and delete buttons? Or should this be a separate issue?

**Comment by fhunleth** [comment]
Yeah, the "new" button should be removed from the user certificate page. User certs only need to be created via the CLI.

Regarding device CA certificates, could you add an issues for creating and deleting them? The issue for creating them depends on #316.

**Comment by pcmarks** [comment]
Referencing https://github.com/nerves-hub/nerves_hub_web/pull/435#pullrequestreview-228364645 discussion.

**Comment by jjcarstens** [comment]
So is this now ready to be closed?
